handlers/main.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
You may reduce this handler by switching `restart logstash necessarily` handler from `service` to `systemd`.
We actually don't have to support old OS versions that are supplied without systemd.

Reply:
Ok, change behaivour